scipy/stats/_binned_statistic.py

```diff
@@ -12,26 +12,28 @@
 
 
 BinnedStatisticResult = namedtuple('BinnedStatisticResult',
-                                   ('statistic', 'bin_edges', 'binnumber'))
+                                   ('statistic', 'bin_edges', 'binnumbers'))
```

Comment:
This is a backward-incompatible change; let's leave the names the same as before so we don't need a deprecation cycle.